cmd/cone/decrypt.go

```diff
@@ -21,7 +21,7 @@ import (
 
 func decryptCredentialCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "decrypt-credential <encrypted-credential>",
+		Use:   "decrypt-credential <app-id>",
```

Comment:
```suggestion
		Use:   "decrypt-credential [app-id]",
```

Optional args are usually denoted by `[]` square brackets